crystalBox.py

```diff
@@ -292,7 +292,22 @@ def dLimits(self,dMin,dMax):
         self.dMin=dMin
         self.dMax=dMax
         self.loadCif()
+        def jasmineFunction(self):
+        print("This is Jasmine's Function")
+    
+    def jasmineFunction(self):
+        print("This is Jasmine's Function")
+        
+    def getEquivalents(self, hkl):
+        sg = SpaceGroupFactory.createSpaceGroup(self.HMSymbol)
+        pg = PointGroupFactory.createPointGroupFromSpaceGroup(sg)
+        
+        equivalents = []
+        equivalents.append(pg.getEquivalents(hkl))
```

Comment:
do u need both lines? couldn't you just merge into:
```
equivalents = pg.getEquivalents(hkl)
```
?

Reply:
good point! Let's see if I can fix it in a way you can see it **without** duplicates this time!!